### PR TITLE
Update AwaLWM2M to 0.2.2

### PR DESCRIPTION
**Pull request by mar3kk**

### CODE REVIEW

File: awalwm2m/Makefile [hunk 1]
```diff
@@ -42,4 +45,4 @@ define Package/awalwm2m/install
 	$(CP) files/etc/* $(1)/etc/
 	$(CP) files/usr/bin/* $(1)/usr/bin/
 endef
-$(eval $(call BuildPackage,$(PKG_NAME)))
+$(eval $(call BuildPackage,$(PKG_NAME)))
```
Comment (abhijit-mahajani-imgtec) [inline awalwm2m/Makefile]: why is this change?

Comment (mar3kk) [inline awalwm2m/Makefile]: Just a typo. Already fixed.

**Comment by abhijit-mahajani-imgtec** [comment]
Approved. Can you now squash both commits?

**Comment by abhijit-mahajani-imgtec** [comment]
Ci40 running awa 0.2.1 not communicating with clicker running awa 0.1.7.

**Comment by mar3kk** [comment]
Updated to AWA 0.2.2

**Comment by abhijit-mahajani-imgtec** [comment]
please change the commit description to 2.2

**Comment by mar3kk** [comment]
updated

File: awalwm2m/Makefile [hunk 2]
```diff
@@ -33,7 +36,7 @@ define Package/awalwm2m/install
 	$(INSTALL_DIR) $(1)/usr/include/
 	$(CP) $(PKG_INSTALL_DIR)/usr/include/* $(1)/usr/include/
 	$(INSTALL_DIR) $(1)/etc/lwm2m
-	$(CP) $(PKG_BUILD_DIR)/core/bootstrap*config $(1)/etc/lwm2m
+	$(CP) $(PKG_BUILD_DIR)/config/*.bsc $(1)/etc/lwm2m
```
Comment (abhijit-mahajani-imgtec) [inline awalwm2m/Makefile]: you also need to replace in the files/etc/init.d/awa_bootstrapd

LOCAL_CONFIG=/etc/lwm2m/bootstrap-contiki.config
FLOW_CONFIG=/etc/lwm2m/bootstrap-contiki-flow-NAT64.config

by 

LOCAL_CONFIG=/etc/lwm2m/contiki.bsc
FLOW_CONFIG=/etc/lwm2m/contiki-flow-NAT64.bsc

That's the reason i was not able to communicate contiki (awa-0.1.7) with Ci40 (awa-0.2.2)

Comment (mar3kk) [inline awalwm2m/Makefile]: Should be fine now

**Comment by abhijit-mahajani-imgtec** [comment]
Approved and Verified

**Pull request merged**